### ISSUE TITLE
500 server error while downloading raspberry os

### ISSUE THREAD

**Issue by ashwin31** [canonical-web-and-design/ubuntu.com#6582]
![Screenshot from 2020-02-06 21-53-02](https://user-images.githubusercontent.com/365610/73956580-24eaa400-492b-11ea-9727-d135c6fcad1f.png)

**Comment by roelofjan-elsinga** [comment]
Try this link instead: https://ubuntu.com/download/raspberry-pi/thank-you?version=18.04.3&architecture=arm64+raspi3

This is related to my issue: https://github.com/canonical-web-and-design/ubuntu.com/issues/6583

**Comment by waldekmi** [comment]
This image doesn't work on Pi 4

**Comment by pmahnke** [comment]
@waldekmi can you give us some more details?  It should work on the 4.  Which Pi do you have?

**Comment by waldekmi** [comment]
I have 4GB Pi4 
Used 64GB SD card, formatted on Win10 with SD Card Formatter, then Fat32Format. Image moved to the card with balenaEtcher.
Pi4 doesn't boot at all (black screen).
Same procedure was applied to burn unofficial images from
https://github.com/TheRemote/Ubuntu-Server-raspi4-unofficial/releases
I tried both Server and Desktop images, they boot but they both have other problems (I described encountered issue there).
Btw, I noticed that your image is called ubuntu-18.04.3-preinstalled-server-arm64+raspi3.img.xz, the same name is under Raspberry 3 and Raspberry 4 buttons. Does it mean it is universal image to be run on Pi3 and Pi4?
